checker/check_runner.go

```diff
@@ -61,19 +60,15 @@ func (l *logger) Debug(desc string, args ...interface{}) {
 	l.messages2 = append(l.messages2, cd)
 }
 
-// UPGRADEv2: to remove.
-func (l *logger) Logf(s string, f ...interface{}) {
-	l.messages = append(l.messages, fmt.Sprintf(s, f...))
-}
-
 func logStats(ctx context.Context, startTime time.Time, result *CheckResult) error {
 	runTimeInSecs := time.Now().Unix() - startTime.Unix()
 	opencensusstats.Record(ctx, stats.CheckRuntimeInSec.M(runTimeInSecs))
 
 	if result.Error != nil {
-		ctx, err := tag.New(ctx, tag.Upsert(stats.ErrorName, scorecarderrors.GetErrorName(result.Error)))
+		ctx, err := tag.New(ctx, tag.Upsert(stats.ErrorName, result.Error2.Error()))
```

Comment:
@azeemsgoogle are you OK with this change? If errors contain temp file name or repo names, it will show as different tags.

Reply:
Oh, I didn't realize `Error2.Error` will print out the whole error string. That will be an issue. Can we use the error mapping logic here: https://github.com/ossf/scorecard/blob/main/errors/names.go.

It'll dump mostly dump out everything as `UnknownError`, but we can improve on it over time. Its better than logging the Error string.